.kitchen.yml

```diff
@@ -40,11 +40,16 @@ platforms:
       image: amazonlinux:2
       platform: rhel
       run_command: /usr/lib/systemd/systemd
+      provision_command:
+        - yum -y install procps-ng
   - name: arch
     driver_config:
       image: archlinux/base
       run_command: /usr/lib/systemd/systemd
       provision_command:
+        - rm -fR /etc/pacman.d/gnupg
+        - pacman-key --init
+        - pacman-key --populate archlinux
         - pacman -Syu --noconfirm --needed systemd grep awk procps which
         - systemctl enable sshd
```

Comment:
There is an issue with generating SSH host keys for Arch in earlier stage.
The command appears to be `ssh-keygen -A -t rsa -f /etc/ssh/ssh_host_rsa_key`.
But, the man page says:
```
If -f has also been specified (with -A), its argument is used as a prefix to the
default path for the resulting host key files.
```
The `-f` parameter is for special use in that case, I don't think we need to specify it.

Reply:
Trying to get why ssh fails to connect. Thought it was the key errors, hence  attempted to insure right keys, but the issue with key errors occurs before any provision_command is run.  Also examining @Ch3LL successful run on her PR, she sees the same issue with key errors but runs successfully, hence removed those commands.  Retrying with my VPN connected to SaltStack but would not expect that to be needed.  The lack of golden images to test against, rather than just latest, bothers me, since the OS can change from one moment to the next: but then it is Arch and that would be a losing battle to keep track of.